minigrid/minigrid_env.py

```diff
@@ -196,6 +196,11 @@ def __str__(self):
 
         output = ""
 
+        # check if self.agent_pos & self.agent_dir is None
```

Comment:
Hey, could we do this is a separate PR and do the full change to use the standard gymnasium implementation? 
I don't think we should hide this away in an unrelated PR

Reply:
Yeah, that's probably better.